networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/GossipTopicNames.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.networking.eth2.gossip.topics;
+
+public class GossipTopicNames {
```

Comment:
I wonder if we can make this an `enum` and then `getAllTopicNames` would be able to iterate through them easily. Would still need special handling for the attestation and sync committee subnets but would at least pick up all the simple topics automatically.

Reply:
Good idea - I'll try to follow-up with this change in a separate PR

Reply:
Follow-up here: https://github.com/ConsenSys/teku/pull/4076